control-plane/tests/test_phase19_operator_workflow_validation.py

```
@@ -1342,6 +1362,91 @@ def get_error_payload(path: str) -> dict[str, object]:
+                                    "message_intent": "Notify the accountable repository owner about the reviewed permission change.",
+                                    "escalation_reason": "Platform administrators must not replace the reviewed analyst request path.",
+                                    "expires_at": (
+                                        datetime.now(timezone.utc) + timedelta(hours=4)
+                                    ).isoformat(),
+                                }
+                            ).encode("utf-8"),
+                            headers={"Content-Type": "application/json"},
+                            method="POST",
+                        ),
+                        timeout=2,
+                    )
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)